### ISSUE TITLE
Åzone Review: Author description error

### ISSUE THREAD

**Issue by aarongillett** [AOzone/median#127]
Both posts read “This is a generic author description”.

**Comment by broskoski** [comment]
Not a bug, maps/author_descriptions.coffee just needs to be updated with copy.

**Comment by aarongillett** [comment]
Perhaps if the field is empty we could omit the value? Not sure if a post from Åzone has a description…

**Comment by broskoski** [comment]
It's in the comps, an author has a description. we can also remove it if
it's not going to be used.

On Saturday, October 24, 2015, aarongillett notifications@github.com
wrote:

> Perhaps if the field is empty we could omit the value? Not sure if a post
> from Åzone has a description…
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/AOzone/median/issues/127#issuecomment-150834607.

## 

**
http://are.na

**Comment by aarongillett** [comment]
Yeah, originally Troy spec’d a description field but maybe for now it’s best to hide it.
